pubsub_nats.go

```diff
@@ -119,14 +119,33 @@ func (p *natsPubSub) Publish(publication *Publication, opts ...PubSubOptPublish)
 
 func (p *natsPubSub) Subscribe(pubs chan *Publication, errors chan error, topic string, opts ...PubSubOptSubscribe) func() {
 
-	config := natsSubscribeConfig{}
+	config := natsSubscribeConfig{
+		replyTimeout: 5 * time.Second,
```

Comment:
If you start needing default, add an initializer

Reply:
Donezo ✅